mono/metadata/object-internals.h

```diff
@@ -1182,11 +1182,18 @@ typedef struct {
 	MonoArray *global_fields;
 	gboolean is_main;
 	MonoArray *resources;
+	GHashTable *unparented_classes;
```

Comment:
You also need to change `mcs/class/corlib/System.Reflection.Emit/ModuleBuilder.cs`